Full-Tortuga/decide-full-tortuga-admin
# Feature/#20 voting form

close #20

## changes
- fix update voting form, added initial values for census, auth and question
- improved the visibility of the statuses and roles

## test
- open the app
- check the voting update form and test it carefully
- check the visual improvements
